etc/ReplayOfflineConfiguration.py

```diff
@@ -240,7 +269,7 @@
                  write_dqm=True,
                  alca_producers=["SiStripPCLHistos", "SiStripCalZeroBias", "TkAlCosmics0T",
                                  "SiPixelCalZeroBias",
-                                 "PromptCalibProdSiStrip", "PromptCalibProdSiPixel"
+                                 "PromptCalibProdSiStrip"
```

Comment:
@francescobrivio isnt the "syntax error" here? What if you have `"SiPixelCalZeroBias", "PromptCalibProdSiStrip"` in the same row?

Reply:
In principle yes, but as you see something is not liked in the syntax check, and this is my best guess on what it could be.